actionview/lib/action_view/railtie.rb

```diff
@@ -39,7 +39,7 @@ class Railtie < Rails::Engine # :nodoc:
 
     initializer "action_view.per_request_digest_cache" do |app|
       ActiveSupport.on_load(:action_view) do
-        if app.config.consider_all_requests_local
+        if app.config.consider_all_requests_local && !Rails.env.test?
           app.executor.to_run ActionView::Digestor::PerExecutionDigestCacheExpiry
```

Comment:
It looks like we could get rid of the `if` entirely, and change this to use `app.reloader.to_run` instead of the executor.

Then it'll automatically get run as part of the reloading in development (if reloading is enabled), and be skipped in test and production (unless reloading is enabled). If it works, that feels like a better statement of our intent -- this current `consider_all_requests_local` check feels more like an indirect attempt to avoid running in production.

Are you up for trying that instead?

(FYI, this won't make 5.0.1 -- as we're in RC stage, the only things that will be added are fixes for regressions relative to 5.0.0. But it does seem a reasonable candidate for backporting for the future 5.0.2.)